index.js

```diff
@@ -28,5 +28,6 @@ module.exports = {
   'QingStor': require('./lib/qingstor/qingstor'),
   'Config': require('./lib/config'),
   'Request': require('./lib/request'),
-  'Signer': require('./lib/sign')
+  'Signer': require('./lib/sign'),
+  'Image': require('./lib/client/image')
```

Comment:
We should export `Client` instead of `Image`.

Reply:
OK